apps/volontulo/templates/auth/login.html

```diff
@@ -3,22 +3,43 @@
 {% block title %}Logowanie do Volontulo{% endblock %}
 
 {% block content %}
-    <h2>Logowanie</h2>
-
-    <form method="post" action="{% url 'login' %}" role="form">
-        {% csrf_token %}
-        <div class="form-group">
-            {{ user_form.email.label_tag }}
-            <input type="text" value="{{ request.POST.email }}" name="email" class="form-control" id="{{ user_form.email.id_for_label }}" />
-        </div>
-        <div class="form-group">
-            {{ user_form.password.label_tag }}
-            <input type="password" value="" name="password" class="form-control" id="{{ user_form.password.id_for_label }}" />
-        </div>
-        <div class="form-group">
-            <a href="{% url 'password_reset' %}">Nie pamiętasz hasła? Możemy pomóc!</a>
-        </div>
-        <input type="hidden" name="next" value="{{ next }}" />
-        <button type="submit" name="submit" class="btn btn-primary">Zaloguj</button>
-    </form>
+    <div class="col-md-5">
+    <h2>Rejestracja w Volontulo</h2>
```

Comment:
we have got a task somewhere for customizable form templates that will support different layout, error messages with proper html and field requirement class names. 
But for now this form layout looks like django default, so it could indeed be used like @tomaszwozniak pointed out